JavaScript/JavaScriptSDK/Telemetry/Common/Envelope.ts

```diff
@@ -29,7 +29,7 @@ module Microsoft.ApplicationInsights.Telemetry.Common {
                     return (this.sampleRate == 100) ? FieldType.Hidden : FieldType.Required;
                 },
                 tags: FieldType.Required,
-                data: FieldType.Required,
```

Comment:
Technically the trailing commas are okay on the last row.  I've always liked not having to add them when adding new rows but it doesn't really matter.